9c-internal/multiplanetary/network/idun.yaml

```diff
@@ -4,14 +4,12 @@ logLevel: "debug"
 global:
   validatorPath: "validator-5.idun.svc.cluster.local"
   networkType: Internal
-  image:
-    repository: "planetariumhq/ninechronicles-headless"
-    tag: "git-9c3d5c20eceb9770bbde41ee66e3d2e005b37e41"
 
   genesisBlockPath: "https://planets-internal.nine-chronicles.com/planets/0x100000000001/genesis"
+  headlessAppsettingsPath: "https://9c-cluster-config.s3.us-east-2.amazonaws.com/9c-internal/heimdall/appsettings.json"
 
   appProtocolVersion: "200071/AB2da648b9154F2cCcAFBD85e0Bc3d51f97330Fc/MEUCIQCjmCK+qUHtgiVfmFIO+uBw.mI4Kpj9T20U4xdIK8I+MgIgSZPL7qAAplhILfRgXwUSDjjfQgQ7A8jgGNqzwDe+h3I=/ZHU5OnRpbWVzdGFtcHUxMDoyMDIzLTA5LTAxZQ=="
-  
+
```

Comment:
https://github.com/planetarium/9c-infra/blob/main/9c-internal/multiplanetary/network/heimdall.yaml#L16-L21? The planet value can be `HeimdallInternal`.